src/autofix/solutions/codeReplacer.ts

```diff
@@ -327,11 +314,110 @@ function patchMessageFixHints(fixHints?: FixHints, apiName?: string) {
 					`$moduleIdentifier.${fnName}(${cleanRedundantArguments(fixHints.exportCodeToBeUsed.args)})`;
 			}
 		}
+	} else if (apiName === "applyTheme" && fixHints?.moduleName === "sap/ui/core/Theming") {
+		if ((fixHints?.exportCodeToBeUsed?.args?.length ?? 0) > 1 &&
+			fixHints?.exportCodeToBeUsed?.args?.[1]?.value !== "undefined") {
+			fixHints = undefined; // We cannot handle this case
+			log.verbose(`Autofix skipped for ${apiName}. Transpilation is too ambiguous.`);
+		}
+	} else if (["attachIntervalTimer", "detachIntervalTimer"].includes(apiName ?? "")) {
+		if ((fixHints?.exportCodeToBeUsed?.args?.length ?? 0) > 1) {
+			fixHints = undefined; // We cannot handle this case
+			log.verbose(`Autofix skipped for ${apiName}. Transpilation is too ambiguous.`);
+		}
+	} else if (apiName === "loadLibrary") {
+		if (fixHints?.exportCodeToBeUsed?.args?.[1]?.kind === SyntaxKind.ObjectLiteralExpression) {
+			const libOptionsExpression =
+				extractKeyValuePairs(fixHints.exportCodeToBeUsed.args[1].value) as {async: boolean; url?: string};
+			if (libOptionsExpression.async === true) {
+				const newArg = {
+					name: fixHints.exportCodeToBeUsed.args[0].value.replace(/^['"]+|['"]+$/g, ""),
```

Comment:
Using back-ticks as first arg (library name) results into quoted backticks.

Reply:
Will be resolved in https://github.com/SAP/ui5-linter/pull/675